### DISCUSSION TITLE
Unable to see content after web portal complete its loading (Found TypeError)

### DISCUSSION THREAD

**Discussion by voyager163** [polymind-inc/acmebot#270] [Q&A] [answered]
1. After webpage complete their loading, the web portal content will disappear. 
2. I have attached the result in this report with the inspect error.

<img width="1263" alt="Screenshot 2021-02-22 at 5 29 09 PM" src="https://user-images.githubusercontent.com/4599055/108688984-7e704180-7533-11eb-8159-37b48f49b1a7.png">

**Comment by shibayan** [comment, marked as answer]
Not a bug. Similar #260 #263

This will happen if you skip Step 4.
https://github.com/shibayan/keyvault-acmebot#4-add-settings-for-your-choice-dns-provider

**Reply by voyager163** [reply]
Hi shibayan,

Thanks for pointing me the correct direction. I miss the configuration portion because it did not know it is one of the steps.

Apologies for the inconvenience cause.

Best Regards,
Jonathan Ng

**Reply by voyager163** [reply]
Hi Shibayan,

Attached is the image that will make it clearer.

### Setup the Application Configuration in the Function

In order to allow the Function to connect to the DNS zone, you are required to add the your Azure DNS subscription ID into the Function.

- `Acmebot:AzureDns:SubscriptionId`
  - Subscription ID is stated in Azure DNS Zone

URL: https://user-images.githubusercontent.com/4599055/108719910-20efeb00-755b-11eb-8b57-f45a0ac6e07c.png

<img width="1280" alt="Screenshot 2021-02-22 at 10 03 10 PM" src="https://user-images.githubusercontent.com/4599055/108719910-20efeb00-755b-11eb-8b57-f45a0ac6e07c.png">
